src/Components/ProductCard.jsx

```diff
@@ -1,28 +1,59 @@
-import React from 'react'
-
-function ProductCard({product}) {
-  return (
-    <div>
-                <div className="vertical-card">
-          <div className="card-img">
-            <img src={product.image} alt="" />
-          </div>
-          <div className="card-text">
-            <h3 className="card-title">{product.name}</h3>
-            <div className="card-price">
-              {product.discountPrice} <span className="card-price-striked">{product.price}</span>
-            </div>
-          </div>
-          <div className="card-btn">
-            <div className="button-box">
-              <a href="" className="btn btn-primary btn-sm">ADD TO CART</a>
-              <i className="fa fa-shopping-cart btn-box-icon btn-sm"></i>
-            </div>
-            <a href="" className="btn btn-tertiary-icon card-btn-icon"><i className="fas fa-heart"></i></a>
-          </div>
-        </div>
-    </div>
-  )
+import React from 'react';
+import { useCart } from '../Context/cart-context';
+function ProductCard({ product }) {
+	const { state, dispatch } = useCart();
+	const checkInCart = () => {
+		for (let i = 0; i < state.cartProducts.length; i++) {
+			if (state.cartProducts[i].id === product.id) {
+				return false;
+			}
+		}
+		return true;
+	};
+	return (
+		<div>
+			<div className="vertical-card">
+				<div className="card-img">
+					<img src={product.image} alt="" />
+				</div>
+				<div className="card-text">
+					<h3 className="card-title">{product.name}</h3>
+					<div className="card-price">
+						{product.discountPrice}{' '}
+						<span className="card-price-striked">{product.price}</span>
+					</div>
+				</div>
+				<div className="card-btn">
+					<div className="button-box">
+						{checkInCart() ? (
+							<a
+								onClick={() => {
+									dispatch({ type: 'ADD_TO_CART', payload: product });
+									console.log(product.inCartQuantity);
+								}}
+								className="btn btn-primary btn-sm"
+							>
+								ADD TO CART
+							</a>
+						) : (
+							<a
+								onClick={() => {
+									console.log('Go TO CART');
+								}}
```

Comment:
remove console.log

Reply:
It's for debugging purposes, I'll remove it once I am done with the basic functionality of cart and wishlist